src/api/utils/fs_utils.ts

```diff
@@ -31,16 +31,24 @@ export function rimraf(thing: string) {
 }
 
 export function copy(from: string, to: string) {
-	if (!fs.existsSync(from)) return;
-
-	const stats = fs.statSync(from);
-
-	if (stats.isDirectory()) {
+	const copied = new Set();
+	const startDir = from + "/";
+	copyFiles(from, to);
+	return copied;
+  
+	function copyFiles(from: string, to: string) {
```

Comment:
Avoid creating this function within another function. If you are going to/need to do something like this, use `const copyFiles = (from:string, to:string) => {}` instead.